backend/src/config/index.ts

```diff
@@ -13,7 +13,7 @@ export default {
       port: process.env.MYSQL_PORT || '3306',
       database: process.env.MYSQL_DATABASE || 'database1',
       username: process.env.MYSQL_USERNAME || 'root',
-      password: process.env.MYSQL_PASSWORD || 'password',
+      password: process.env.MYSQL_PASSWORD || '',
```

Comment:
Remove this change, as from previous experience, this causes issues for automated tests

Reply:
what is the meaning of removing the change?

Reply:
The default `password` is expected by the automated tests.